docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
I'm not sure sure I understand. Why would anyone get the idea to put other things into that `.env` file since it won't ever make it to deploy? (Unless people deploy out of git repos which comes with its own set of issues).